zerver/lib/test_helpers.py

```diff
@@ -70,13 +72,15 @@ def tornado_redirected_to_list(lst):
 
 @contextmanager
 def simulated_empty_cache():
-    # type: () -> Generator[List[Tuple[str, str, str]], None, None]
-    cache_queries = []
+    # type: () -> Generator[List[Tuple[str, Union[str, List[str]], str]], None, None]
+    cache_queries = []  # type: List[Tuple[str, Union[str, List[str]], str]]
```

Comment:
We've been doing one space before the `#` in our type annotations, can you update this in your code?